Mathlib/Algebra/Module/Submodule/Localization.lean

```diff
@@ -113,3 +118,50 @@ instance IsLocalizedModule.toLocalizedQuotient' (M' : Submodule R M) :
 
 instance (M' : Submodule R M) : IsLocalizedModule p (M'.toLocalizedQuotient p) :=
   IsLocalizedModule.toLocalizedQuotient' _ _ _ _
+
+section LinearMap
+
+variable {P : Type w} [AddCommGroup P] [Module R P]
+variable {Q : Type w'} [AddCommGroup Q] [Module R Q] [Module S Q] [IsScalarTower R S Q]
+variable (f' : P →ₗ[R] Q) [IsLocalizedModule p f']
+
+lemma LinearMap.localized'_ker_eq_ker_localizedMap (g : M →ₗ[R] P) :
+    Submodule.localized' S p f (LinearMap.ker g) =
+      LinearMap.ker ((IsLocalizedModule.map p f f' g).extendScalarsOfIsLocalization p S) := by
```

Comment:
Can you provide a version `LinearMap.ker (IsLocalizedModule.map p f f' g) = ((LinearMap.ker g).localized' S p f).restrictScalars _`?